jans-linux-setup/jans_setup/setup_app/utils/package_utils.py

```
@@ -69,20 +69,14 @@ def check_and_install_packages(self):
-                                        'mysql-community-server-core_9.2.0-1debian12_amd64.deb',
-                                        'mysql-client_9.2.0-1debian12_amd64.deb',
-                                        'mysql-community-client_9.2.0-1debian12_amd64.deb',
-                                        'mysql-community-client-core_9.2.0-1debian12_amd64.deb',
-                                        'mysql-community-client-plugins_9.2.0-1debian12_amd64.deb',
-                                        ):
-                            base.download(f'https://downloads.mysql.com/archives/get/p/23/file/{deb_fn}', os.path.join(tmpdirname, deb_fn))
-                        self.run([install_command.format(os.path.join(tmpdirname, '*.deb'))],  shell=True)
+                        apt_deb_fn = 'mysql-apt-config_0.8.39-1_all.deb'
+                        local_apt_path = os.path.join(tmpdirname, apt_deb_fn)
+                        base.download(f'https://repo.mysql.com//{apt_deb_fn}', local_apt_path)
+                        self.run([install_command.format(local_apt_path)],  shell=True)
```

Ruff (0.15.11)

[error] Function call with `shell=True` parameter identified, security issue

(S604)